src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -55,4 +55,11 @@ Feature: Vodqa test
   Scenario: Validate that user is able to scroll vertically by screen percentage
     Given I login to vodqa application using valid credentials
     When I scroll vertically from 60 percent height to 20 percent height and 50 percent width
-    Then Element text "Jasmine" should be visible
\ No newline at end of file
+    Then Element text "Jasmine" should be visible
+
+    #  CONFIG=./configs/vodqa_local_config.properties TAG=@dragAndDrop PLATFORM=android ./gradlew run
+  @android @dragAndDrop
+  Scenario: Validate that user is able to drag and drop
+    Given I login to vodqa application using valid credentials
+    When I drag the drag me circle object to drop here target
```

Comment:
It looks wierd,
it could be something like - 
When I drag the circle to drop here target